console/src/assets/i18n/en.json

```diff
@@ -629,7 +629,7 @@
         "TIMEOUT_DESCRIPTION": "Set the maximum time your target has to respond. If it takes longer, we will stop the request.",
         "INTERRUPT_ON_ERROR": "Interrupt on Error",
         "INTERRUPT_ON_ERROR_DESCRIPTION": "Stop all executions when the targets returns with an error",
-        "INTERRUPT_ON_ERROR_WARNING": "Caution: “Interrupt on Error” halts operations on failure, risking lockout. Test with it disabled to prevent blocking login/creation.",
+        "INTERRUPT_ON_ERROR_WARNING": "Caution: "Interrupt on Error" halts operations on failure, risking lockout. Test with it disabled to prevent blocking login/creation.",
```

Comment:
You should revert back to the other type of quotes, it's breaking the build process.

Other option is to escape the double quotes

Reply:
@IAM-marco 
Thanks, I chose reverting one. [b0fe004](https://github.com/zitadel/zitadel/pull/9823/commits/b0fe004c83ef9f549de442226f6011bc4492387c)